libraries/SD_MMC/src/SD_MMC.cpp

```diff
@@ -244,7 +245,7 @@ bool SDMMCFS::begin(const char *mountpoint, bool mode1bit, bool format_if_mount_
     .cd = SDMMC_SLOT_NO_CD,
     .wp = SDMMC_SLOT_NO_WP,
     .width = 4,
-    .flags = 0,
+    .flags = SDMMC_SLOT_FLAG_UHS1,
```

Comment:
you should probably check here if `SOC_SDMMC_UHS_I_SUPPORTED` also. Seems undefined in IDF 5.3